postgrest/_sync/request_builder.py

```diff
@@ -156,6 +156,7 @@ def upsert(
         count: Optional[CountMethod] = None,
         returning: ReturnMethod = ReturnMethod.representation,
         ignore_duplicates: bool = False,
+        on_conflict: str = "",
```

Comment:
This new arg has to be documented in this method's docstrings so that it shows up on our autodocs

Reply:
Done